web-components/src/components/tabs/Tabs.ts

```diff
@@ -274,7 +277,29 @@ export namespace Tabs {
       this.tabHiddenIdPositiveTabIndex = hiddenTab ? hiddenTab.id : undefined;
     }
 
+    /* Sort tabs and panes as per the user prefs saved in the localStorage */
+    private sortTabsAndPanes() {
+      if (!this.tabsOrderPrefsArray.length) {
+        return;
+      }
+      const comparator = (a: any, b: any) => {
```

Comment:
IS it possible to avoid use of **any**?

Reply:
since the comparator is within the scope of calling functions parent. `any` type is better in terms of code readability and avoiding unwanted default values.

Reply:
addressed